test/powershell/Modules/Microsoft.PowerShell.Utility/Add-Type.Tests.ps1

```diff
@@ -36,33 +36,33 @@ Describe "Add-Type" -Tags "CI" {
     }
 
     It "Public 'Language' enumeration contains all members" {
-        [Enum]::GetNames("Microsoft.PowerShell.Commands.Language") -join "," | Should Be "CSharp,VisualBasic"
+        [Enum]::GetNames("Microsoft.PowerShell.Commands.Language") -join "," | Should -Be "CSharp,VisualBasic"
     }
 
     It "Should not throw given a simple class definition" {
-        { Add-Type -TypeDefinition "public static class foo { }" } | Should Not Throw
+        { Add-Type -TypeDefinition "public static class foo { }" } | Should -Not -Throw
     }
 
     It "Can use System.Management.Automation.CmdletAttribute" {
         $code = @"
 [System.Management.Automation.Cmdlet("Get", "Thing", ConfirmImpact = System.Management.Automation.ConfirmImpact.High, SupportsPaging = true)]
 public class AttributeTest$guid {}
 "@
-        Add-Type -TypeDefinition $code -PassThru | Should Not Be $null
+        Add-Type -TypeDefinition $code -PassThru | Should -Not -BeNullOrEmpty
     }
 
     It "Can load TPA assembly System.Runtime.Serialization.Primitives.dll" {
-        Add-Type -AssemblyName 'System.Runtime.Serialization.Primitives' -PassThru | Should Not Be $null
+        Add-Type -AssemblyName 'System.Runtime.Serialization.Primitives' -PassThru | Should -Not -BeNullOrEmpty
     }
 
     It "Can compile C# files" {
 
-        { [Test.AddType.BasicTest1]::Add1(1, 2) } | Should Throw
-        { [Test.AddType.BasicTest2]::Add2(3, 4) } | Should Throw
+        { [Test.AddType.BasicTest1]::Add1(1, 2) } | Should -Throw
```

Comment:
We should never have just `Should -Throw`, please get the error being thrown and add `-ErrorId`.  We want to prevent the test from passing if it throws for a different reason than what is being tested (like out of memory, for example).